skyportal/tests/data/aligo_O4high.txt

```diff
@@ -0,0 +1,2736 @@
+1.021659e+01 6.662774e-21
```

Comment:
Can we name this something more informative (the words "noise" and "spectrum" should be up there with "LIGO").

Reply:
Yeah added!